kiva/fonttools/font_manager.py

```diff
@@ -1440,18 +1497,21 @@ def findfont(prop, fontext='ttf'):
         return result
 
 else:
-    try:
-        fontManager = pickle_load(_fmcache)
-        if (not hasattr(fontManager, '_version') or
-                fontManager._version != FontManager.__version__):
-            _rebuild()
-        else:
-            fontManager.default_size = None
-            logger.debug("Using fontManager instance from %s", _fmcache)
-    except Exception:
-        _rebuild()
 
     def findfont(prop, **kw):
-        global fontManager
-        font = fontManager.findfont(prop, **kw)
+        font = default_font_manager().findfont(prop, **kw)
         return font
```

Comment:
Opened https://github.com/enthought/enable/issues/497
(Part of me wonders if this needs a deprecation warning or if we are confident no one uses this enough it can just die.)

Reply:
Thanks. I noticed some confusion between `findfont` and `findFont` in that issue, but the matter is settled here.

Reply:
Well spotted. Will fix the spelling on the issue.